scripts/factory-hooks/post-site-install/post-site-install.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Factory Hook: post-site-install
+#
+# This is used so that an ACSF site install is identical to the local BLT site
+# install, with the environment, site, and uri CLI runtime arguments overriding
+# all other configuration.
+#
+# Usage: SCRIPTNAME site env db-role domain
+
+# Exit immediately on error and enable verbose log output.
+set -ev
```

Comment:
'set -e' is not doing anything in this script. Then again 1) it isn't doing anything in db-update.sh either 2) it feels like a good thing to do in general. So.